README.md

```diff
@@ -73,34 +73,74 @@ C# · Go · Java · JavaScript · JSX · JSON · PHP · Python · Ruby · Scala
 See <a href="https://semgrep.dev/docs/supported-languages/">supported languages</a> for the complete list.
 </p>
 
-### Getting started
+### Getting started (Recommended)
 
-To install Semgrep use Homebrew or pip, or run without installation via Docker:
+1.  Register to <a href="http://bit.ly/424Jbs0" target="_blank">semgrep.dev</a>
+<img src="images/semgrep-main-image.jpg" /> 
 
-```sh
-# For macOS
-$ brew install semgrep
 
-# For Ubuntu/WSL/Linux/macOS
-$ python3 -m pip install semgrep
+2.  Install Semgrep CLI
 
-# To try Semgrep without installation run via Docker
-$ docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep
-```
+  ```
+  # For macOS
+  $ brew install semgrep
 
-Once installed, Semgrep can run with single rules or entire rulesets. Visit [Docs > Running rules](https://semgrep.dev/docs/running-rules/) to learn more or try the following:
+  # For Ubuntu/WSL/Linux/macOS
+  $ python3 -m pip install semgrep
 
-```sh
-# Check for Python == where the left and right hand sides are the same (often a bug)
-$ semgrep -e '$X == $X' --lang=py path/to/src
+  # To try Semgrep without installation run via Docker
+  $ docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep
+  ```
+
+3.  Run `semgrep login` to get the login URL. Open the login URL in the browser and login.
+
+<img src="images/semgrep-login.jpg" /> 
+
+4. Go to your app's root directory and run `semgrep --config auto`
+
+<img src="images/semgrep-auto-config-example.jpg" /> 
+
+
+### Getting started (CLI only)
+In this flow, you'll lose the simplicity of quickly seeing and managing results and setting up CI/CD pipelines with minimal effort. Additionally, you'll be unable to use additional proprietary Semgrep Registry rules and run custom policies from the Semgrep App.
+
+ 1. Install Semgrep CLI
+
+  ```
+  # For macOS
+  $ brew install semgrep
+
+  # For Ubuntu/WSL/Linux/macOS
+  $ python3 -m pip install semgrep
 
-# Fetch rules automatically by setting the `--config auto` flag.
-# This will fetch rules relevant to your project from Semgrep Registry.
-# Your source code is not uploaded.
-$ semgrep --config auto
+  # To try Semgrep without installation run via Docker
+  $ docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep
+  ```
+
+2. Go to your app's root directory and run the following:
+```
+semgrep --config auto
 ```
 
-To run Semgrep Supply Chain, [contact the Semgrep team](https://semgrep.dev/contact-us).
+### Set up CI/CD in two minutes
+
+1.  Register or login to <a href="http://bit.ly/424Jbs0" target="_blank">semgrep.dev</a>
+
+
+2. Go to `Projects > Scan New Project > Run scan in CI`
+<img src="images/semgrep-ci-cd-popup.jpg" /> 
+
+
+3. Select your version control system and follow the wizard to add your project. After this setup, you'll get continuous security. Semgrep will scan your project after every pull request!
```

Comment:
I'm not sure about this line: "you'll get continuous security" (our rules / support vary per language) - I wonder if there's a more accurate way to explain the value add?